Fixing Edit Event functionality

CoderMayhem opened the pull request:
<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->

**What kind of change does this PR introduce?**
This PR fixes the faulty edit event feature and makes it functional. 

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Did you add tests for your changes?**
No

**Summary**
Fixes #525

Sagar2366 commented:
@CoderMayhem seems like issue is still not fixed. 
Please share solution video too. Also, on saving updated event, bottom nav bar disables/remove.

CoderMayhem commented:
Okay. I'll look into this.

Sagar2366 commented:
@CoderMayhem disable code formatter and send only changes required to fix this issue.

CoderMayhem commented:
Sure, sorry for the trouble.

Sagar2366 reviewed (commented):
> Sure, sorry for the trouble.

resolve conflicts on this PR.

CoderMayhem commented on lib/views/pages/login_signup/set_url_page.dart:
```diff
@@ -75,9 +75,9 @@ class _UrlPageState extends State<UrlPage> with TickerProviderStateMixin<UrlPage
     setState(() {
       orgUrl =
-          "${dropdownValue.toLowerCase()}://${urlController.text}/talawa/graphql/";
+          "${dropdownValue.toLowerCase()}://${urlController.text}/graphql/";
```
Done

CoderMayhem commented:
@Sagar2366 sir can you please review this at your convenience. I have removed most of the unnecessary format changes and fixed the delay problem. Right now my exams are going on so I am bit occupied. I'll open an issue for showing the toast (when user tries to edit an event he/she did not create) once my exams are over.

Sagar2366 commented:
@CoderMayhem seems like issue is still not fixed, please verify it again.
Editing event does not reflect changes on events page.

CoderMayhem commented:
https://user-images.githubusercontent.com/56952811/114311943-72096d80-9b0e-11eb-9b10-b2beccd921ae.mp4

@Sagar2366 sir the events are getting edited,  **but you can edit only those which you have created**. So please try to create an event and then try editing it. I'll be updating the issue description and will add the feature to stop users from editing events they did not create in this PR itself.

CoderMayhem commented:
@Sagar2366 the suggested changes have been made. Kindly review at your convenience.
